Formula/libgpg-error.rb

```diff
@@ -25,6 +25,7 @@ def install
                           "--prefix=#{prefix}",
                           "--enable-static"
     system "make", "install"
+    bin.install "src/gpg-error-config"
```

Comment:
It looks like `make install` no longer installs this. Which is weird, because the build configures `gpg-error-config.in`.

We should check with upstream whether this was intentional.

Reply:
Yep, looks intentional. https://dev.gnupg.org/T5683